### PR TITLE
Updated AdMob Integrations

### PR DESCRIPTION
**Pull request by pinkeshmars**
### Description
Updated AdMob Integrations

Linear ticket and [magic word](https://linear.app/docs/github#link-prs) 
Fixes [DEVR-357](https://linear.app/flutterflow/issue/DEVR-357/integration-ads) 
Fixes [DEVR-630 ](https://linear.app/flutterflow/issue/DEVR-630/admob-consent-dialog-setting)

 
## Type of change
- [ ] Typo fix 
- [ ] New feature 
- [x] Enhancement to current docs
- [x] Removed outdated references
- [x] Update assets

### CODE REVIEW

File: docs/ff-integrations/ads/admob.md [hunk 1]
```diff
@@ -101,43 +84,50 @@ To add the APP keys:
 </div>
 <p></p>
 
-## Showing ads
+### 3. Configure optional settings
 
-After adding the keys, you can find the instructions to display the ad on below pages:
+Below are some AdMob settings (under **Settings and Integrations** menu) that you might need to configure based on your app and target audience.
```
Comment (PoojaB26) [inline docs/ff-integrations/ads/admob.md:89]: A screenshot from Settings page would have been useful.

File: docs/ff-integrations/ads/admob.md [hunk 2]
```diff
@@ -101,43 +84,50 @@ To add the APP keys:
...
 :::
+
 <div style={{
     position: 'relative',
     paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
     height: 0,
     width: '100%'}}>
     <iframe 
-        src="https://demo.arcade.software/PBkC8jgpGHT2zrPhl5yh?embed&show_copy_link=true"
+        src="https://demo.arcade.software/vlm11muoQuU7YRS2AzQW?embed&show_copy_link=true"
```
Comment (PoojaB26) [inline docs/ff-integrations/ads/admob.md:130]: In this Arcade, there is an info about ad banners having a default dimension of X. I think that info should exist out of Arcade as an admonition .

File: docs/ff-integrations/ads/admob.md [hunk 3]
```diff
@@ -101,43 +84,50 @@ To add the APP keys:
...
+Once the setup is completed, you can start to display [AdBanner](#adbanner) or [Interstitial ads](#interstitial-ad) in your app.
 
-1. [Adding AdBanner widget](#1-adding-adbanner-widget)
-2. [Assigning Ad Unit ID](#2-assigning-ad-unit-id)
-3. [Enabling test ads](#3-enabling-test-ads)
-4. [Testing ads](#4-testing-ads)
+## AdBanner
 
-### 1. Adding AdBanner widget
+The **AdBanner** widget displays advertisement banners within your app. It can feature text, images, and rich media, including video ads.
```
Comment (PoojaB26) [inline docs/ff-integrations/ads/admob.md:110]: Any example image that includes adbanners in the ecommerce app?

File: docs/ff-integrations/ads/admob.md [hunk 4]
```diff
@@ -427,53 +279,22 @@ To load the ad on page load:
...
 <p></p>
 
-### 4. Display Ad
+## Best Practices
 
-In this step, you will show the ad plus check if a user has dismissed the ad. If so, you will load the new ad and then move to the next page.
+To maximize the effectiveness of AdMob ads in your app while maintaining a positive user experience and complying with AdMob policies, follow these overall best practices:
 
-The steps to display an interstitial ad are as follows:
+1. **Use Test Ads During Development**: Always enable Test Ads during development to avoid invalid traffic and protect your AdMob account from being flagged or banned.
```
Comment (PoojaB26) [inline docs/ff-integrations/ads/admob.md:286]: They should be bullet points and not numbered. Fix for all the best practices below this.

**Review by PoojaB26 — CHANGES_REQUESTED** [review]
Need few fixes. 

Also, I know we mentioned about fixing the looping feature of Arcades? Like stopping the Arcade at the end of the slides and let users restart it. I see all the arcades here are looped as soon as the last slide is clicked. If we found any solution, can we apply them here?

**Comment by PoojaB26** [comment]
Hey @pinkeshmars it seems the diagrams have started using pointed arrows instead of bullet arrows, can you please fix the arrows and to maintain consistency, not use pointed arrows anywhere (other than very special circumstances like flowcharts). I see this being used more lately, please go to Figma and check your last few diagrams and fix them and update the files in this PR (even if its not admob related, for example for new feature release docs)

**Comment by pinkeshmars** [comment]
> Need few fixes.
> 
> Also, I know we mentioned about fixing the looping feature of Arcades? Like stopping the Arcade at the end of the slides and let users restart it. I see all the arcades here are looped as soon as the last slide is clicked. If we found any solution, can we apply them here?

Yes, we initially enabled the stepper to help users track their progress, so if they start from the first step, they’ll have a clear indication of where they are. However, it’s not a perfect solution. After exploring more options in Arcade, I found that we can resolve this by setting the hotspot's 'Destination' to 'This Step'. All arcades on this page are fixed now.

Please note that this issue only occurs when the last step contains a hotspot. We’ll need to apply this solution for all the arcades. We can discuss further how we should approach it.

> Hey @pinkeshmars it seems the diagrams have started using pointed arrows instead of bullet arrows, can you please fix the arrows and to maintain consistency, not use pointed arrows anywhere (other than very special circumstances like flowcharts). I see this being used more lately, please go to Figma and check your last few diagrams and fix them and update the files in this PR (even if its not admob related, for example for new feature release docs)

Sure, other images are for localization, which is WIP, and streaming Supabase query, which is not merged yet, so I shall update in that PR only.

File: docs/ff-integrations/ads/admob.md (hunk 2 above)
Comment (pinkeshmars) [reply docs/ff-integrations/ads/admob.md:130]: ok, added into info admonition.

File: docs/ff-integrations/ads/admob.md (hunk 3 above)
Comment (pinkeshmars) [reply docs/ff-integrations/ads/admob.md:112]: Yes, added one inside the tip admonition.

File: docs/ff-integrations/ads/admob.md (hunk 4 above)
Comment (pinkeshmars) [reply docs/ff-integrations/ads/admob.md]: ok, fixed.

**Comment by pinkeshmars** [comment]
@PoojaB26 this is ready to review again.

**Review by PoojaB26 — APPROVED** [review]

**Pull request merged**